### PR TITLE
Improvements/navigation

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/qU7xWbYP/788-update-editor-navigation)

## Context

This PR adds new navigation/layout to the editor.

## Before

Screenshot 2020-07-27 at 12 03 05

## After

Screenshot 2020-07-27 at 16 27 08

Screenshot 2020-07-27 at 16 27 19

Screenshot 2020-07-27 at 16 27 36
Screenshot 2020-07-27 at 16 26 43

Screenshot 2020-07-27 at 16 27 50

## What needs to be finished

- [x] Show to Kellie
- [x] Talk with Kellie about the buttons "Edit | Preview" which overlaps the current navigation. I moved more to the bottom but I don't know the expectation.
- [x] Review content

### CODE REVIEW

File: templates/nunjucks/base.njk.html [hunk 1]
```diff
@@ -53,6 +53,9 @@
           Ministry of Justice
         </a>
 
+        <a class="moj-header__link moj-header__link--service-name"
+        href="{{ flowUrl }}">MOJ Online Editor</a>
```
Comment (brenetic) [inline templates/nunjucks/base.njk.html:57]: Are we using `MoJ Online Editor` now or are we keeping `Form Builder Console`?

Comment (tomas-stefano) [reply templates/nunjucks/base.njk.html:57]: We are using MoJ Online Editor

Comment (brenetic) [reply templates/nunjucks/base.njk.html:57]: okey dokey. if that's the case we should make that consistent across our platform and tools

**Comment by tomas-stefano** [comment]
Closing this PR because CircleCI had an outage that for some reason blocked the CI to continue. Closing in favor of #276

**Pull request closed without merging**